### PR TITLE
Multi language support functionality is added

### PR DESCRIPTION
**Pull request by mohammra**
With the added functionality user can select desired language in the begining

### CODE REVIEW

File: Voice Note/ViewModels/SpeechRecognizer.swift [hunk 1]
```diff
@@ -79,45 +76,13 @@ class SpeechRecognizer: ObservableObject {
...
-     This method removes the speech transcription task
-     */
+
     func reset() {
         task?.cancel()
         task = nil
     }
 }
-
-extension SFSpeechRecognizer {
```
Comment (anwar-ulhaq) [inline Voice Note/ViewModels/SpeechRecognizer.swift:99]: @giaongo was not this important?

**Review by anwar-ulhaq — APPROVED** [review]
@giaongo also should approve as i do not have much understanding to Speech feature

File: en.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* 
+  Localizable.strings
+  Voice Note
+
+  Created by Mohammad Rashid on 30.4.2023.
+  
+*/
```
Comment (giaongo) [inline en.lproj/Localizable.strings:7]: What does this file do? @mohammra

**Review by giaongo — APPROVED** [review]
Everything is OK! Please merge @anwar-ulhaq

File: en.lproj/Localizable.strings (hunk 2 above)
Comment (mohammra) [reply en.lproj/Localizable.strings:7]: This helps to localise the language

**Pull request merged**